sky/provision/oci/query_utils.py

```diff
@@ -21,6 +21,8 @@
 from sky.adaptors import common as adaptors_common
 from sky.adaptors import oci as oci_adaptor
 from sky.clouds.utils import oci_utils
+from sky.exceptions import ResourcesUnavailableError
```

Comment:
```suggestion
from sky import exceptions
```
nit: import module instead of class/functions
https://google.github.io/styleguide/pyguide.html#224-decision